js/components/simulation.tsx

```diff
@@ -41,6 +41,27 @@ export default class Simulation extends BaseComponent<IBaseProps, IState> {
       </div>);
   }
 
+  getDialogOffset() {
+    const { selectedBoundary } = this.simulationStore;
+    const { screenPos } = selectedBoundary || {};
+    const topBarHeight = 20;
+    const dialogSize = { width: 250, height: 150 };
+    const dialogOffset = { x: 150, y: -100 };
+    if (screenPos) {
+      const planetWizardElt = document.querySelector(".planet-wizard");
```

Comment:
This seems a bit fragile and I think this could be replaced with `refs`.

Reply:
Good point -- I'll look into it.

Reply:
The presence of jQuery in some of the coordinate conversion code got me thinking in DOM element terms rather than React terms. Fixed